client/components/orderable-list/style.scss

```diff
@@ -24,7 +24,12 @@
 	}
 
 	&__drag-handle {
+		&#{&} {
```

Comment:
What does this line do?

Reply:
It basically repeats the class name in the compiled CSS to increase the specificity of the rule without having to use `!important`